pkg/freeze/containerd/pause.go

```diff
@@ -89,27 +92,31 @@ func lookupContainerID(ctx context.Context, conn *grpc.ClientConn, podUID, conta
 		},
 	})
 	if err != nil {
-		return "", err
+		return []string{}, err
 	}
 
 	if len(pods.Items) == 0 {
-		return "", fmt.Errorf("pod %s not found", podUID)
+		return []string{}, fmt.Errorf("pod %s not found", podUID)
 	}
 	pod := pods.Items[0]
 
 	ctrs, err := client.ListContainers(ctx, &cri.ListContainersRequest{Filter: &cri.ContainerFilter{
 		PodSandboxId: pod.Id,
-		LabelSelector: map[string]string{
-			"io.kubernetes.container.name": containerName,
-		},
 	}})
 	if err != nil {
-		return "", err
+		return []string{}, err
+	}
+
+	var ids []string
+	for _, c := range ctrs.Containers {
+		if c.GetMetadata().GetName() != "queue-proxy" {
+			ids = append(ids, c.Id)
+		}
 	}
 
-	if len(ctrs.Containers) == 0 {
-		return "", fmt.Errorf("container %q in pod %q not found", containerName, podUID)
+	if len(ids) == 0 {
+		return []string{}, fmt.Errorf("no non queue-proxy containers found in pod %q", podUID)
```

Comment:
nit, but this can be `nil,  fmt.Errorf("no non queue-proxy containers found in pod %q", podUID)`